controllers/client_main.php

```diff
@@ -129,6 +129,16 @@ public function index()
         }
     }
 
+    /**
+     * Client widget
+     */
+    public function widget()
+    {
+        $this->action = 'index';
```

Comment:
The view needs to be updated with links to /widget when loaded via this method, otherwise status and sorting links won't work in the widget when the navigation action is disabled.